Gemfile

```diff
@@ -28,7 +28,7 @@ gem "redis-namespace", "~> 1.5.2"
 gem "statsd-ruby", "~> 1.3.0"
 
 gem "plek", "~> 1.12"
-gem "gds-api-adapters", "~> 30.0"
+gem "gds-api-adapters", github: "alphagov/gds-api-adapters"
```

Comment:
This looks like a dangerous move to me, if someone merges something to master that later reveals to be broken (not caught by the tests) we might have problems.
Or even if something goes to master a few seconds before we deploy, we might have some unexpected behaviour.